.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "stable"
```

Comment:
You need to have a `script` in order to properly use Travis CI later. Add the command `npm run linting` and `npm test` and take [this file](https://github.com/julmot/mark.js/blob/master/.travis.yml) as an example.